### DISCUSSION TITLE
Password disclosed in logs of Git based WIKI Single File Updater

### DISCUSSION THREAD

**Discussion by markbangert** [rfennell/AzurePipelines#943] [General]
Hi, we am using your tools to automatically generate release notes. Great work. Thank you so much. This really makes a lot of my administrative tasks around the release management easier.

Today we saw in the logs of this task that there is a password disclosed:

![image](https://user-images.githubusercontent.com/10157181/107566593-5b659980-6be5-11eb-8a75-505a07dbbca0.png)
(highlighted in green - sorry for the mess but I had to black out some other stuff)

Where does this passwork come from? Do I have to worry that this is visible in the logs? Any chance to hide it?

Thank you in advance for your help, kind regards, Mark

**Comment by rfennell** [comment]
Good catch, thanks for reporting it. I have just published a revised version that addresses the logging issue

**Comment by markbangert** [comment]
Works! Thank you so much for rolling out a fix so quickly...

![image](https://user-images.githubusercontent.com/10157181/107608401-29335680-6c3c-11eb-9b70-638aaf2d7890.png)
